instance.tf

```diff
@@ -47,9 +57,9 @@ resource "aws_instance" "fsb_instance" {
 }
 
 #CREATE SG
-resource "aws_security_group" "FSB_SG" {
+resource "aws_security_group" "My_sg" {
   name_prefix = var.SGname2
-  vpc_id      = aws_vpc.fsb_vpc.id
+  vpc_id      = "vpc-0ca70b4fc5a1ecdca"
```

Comment:
Could you please avoid hardcoding vpc id but rather use resource reference like you did before.